### PR TITLE
Clear up some of the secrets usage.

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,38 +77,47 @@ Another purpose is to serve as an example of how to build an Event Source using
...
    1. Give Google Cloud Storage permissions to publish to GCP Pub Sub.
-      1. First find the Service Account that GCS uses to publish to Pub Sub (Either using UI, or using curl as shown below)
-	     1. Use the [Cloud Console or the JSON API](https://cloud.google.com/storage/docs/getting-service-account)
-            Assume the service account you found from above was `service-XYZ@gs-project-accounts.iam.gserviceaccount.com`, you'd do:
-			```shell
-			export GCS_SERVICE_ACCOUNT=service-XYZ@gs-project-accounts.iam.gserviceaccount.com
-			```
+
+      1. First find the Service Account that GCS uses to publish to Pub Sub
+         (Either using UI, or using curl as shown below) 1. Use the
```
Comment (vaikas-google) [inline README.md]: this 1. Use the section seems wrongly formatted.

File: README.md [hunk 2]
```diff
@@ -110,32 +126,31 @@ Another purpose is to serve as an example of how to build an Event Source using
...
-      ```
+1. Create two secrets on the kubernetes cluster with the downloaded key:
 
-      The name `gcssource-key` and `key.json` are pre-configured values
-      in the controller which manages your Cloud Storage sources.
+   ```shell
+   # Note that the first secret may already have been created when installing
+   # Knative Eventing. The following command will overwrite it. If you don't
+   # want to overwrite it, then skip this command.
+   kubectl --namespace gcssource-system create secret generic gcs-source-key --from-file=key.json=gcs-source.json --dry-run --output yaml | kubectl apply --filename -
```
Comment (vaikas-google) [inline README.md]: why change this like so? in case it already exists?

Comment (vaikas-google) [reply README.md]: This is the gcs-source key so I don't see this actually existing. I can see the one below existing if pubsub has already been created?

File: README.md [hunk 3]
```diff
@@ -370,22 +397,22 @@ gcloud projects add-iam-policy-binding $PROJECT_ID \
...
 kubectl delete services.serving gcs-message-dumper
 ```
 
 # **REST OF THIS DOCUMENT NEEDS UPDATING**
 
-
 ## More complex examples
-* [Multiple functions working together](MULTIPLE_FUNCTIONS.md)
 
+- [Multiple functions working together](MULTIPLE_FUNCTIONS.md)
```
Comment (vaikas-google) [inline README.md]: Just remove the rest here or better yet, add a quick sample sinking these into a broker :)

File: README.md (hunk 2 above)
Comment (n3wscott) [reply README.md]: I copied it from the updated docs in knative/docs: https://github.com/knative/docs/blob/8d62a0b0663a8f2b91e174f805b1837b6ec108ec/docs/eventing/samples/gcp-pubsub-source/README.md

Comment (n3wscott) [reply README.md]: I see what you are saying, yeah that needs to be cleaned.. one sec

File: README.md (hunk 3 above)
Comment (n3wscott) [reply README.md]: Deleted and added broker.

File: README.md (hunk 2 above)
Comment (n3wscott) [reply README.md]: reverted it to just create the key

**Pull request merged**